e2e-tests/development-runtime/cypress/integration/hot-reloading/error-handling/overlay-endpoints.js

```diff
@@ -0,0 +1,11 @@
+const cwd = Cypress.config(`projectRoot`)
+
+describe(`overlay handlers don't serve unrelated files`, () => {
+  it(`__file-code-frame`, () => {
+    cy.request(
+      `__file-code-frame?filePath=${cwd}/SHOULD_NOT_SERVE&lineNumber=0`
+    ).should(response => {
+      expect(response.body.codeFrame).not.to.match(/CYPRESS-MARKER/)
+    })
+  })
```

Comment:
v4 doesn't have the other endpoint (`__original-stack-frame` with `skipSourceMap` param) that would serve content outside of.

We probably should streamline this in v5 to have single handler instead of 2 the basically do same thing just used in different places, but that's outside of scope of this